raw-spans-grouper/helm/values.yaml

```diff
@@ -112,6 +112,9 @@ rawSpansGrouperConfig:
         bootstrapServers: "bootstrap:9092"
         schemaRegistryUrl: "http://schema-registry-service:8081"
         cacheMaxBytesBuffering: 16777216 # 16MB
+        commitIntervalMs: 5000 #default
```

Comment:
Hi @samarth-gupta-traceable would you mind explaining why this value is a good default? Why is this better than 10000 or 2500 for example?